package.json

```diff
@@ -45,6 +45,7 @@
     "bs58": "^4.0.1",
     "crypto-addr-codec": "^0.1.7",
     "nano-base32": "^1.0.1",
-    "ripemd160": "^2.0.2"
+    "ripemd160": "^2.0.2",
+    "thor-devkit": "^1.3.3"
```

Comment:
If you included external package, please commit `package-lock.json` as well

Reply:
`Please also report how much the file increased by running npm run size. If it increased the file significantly we will park the PR and will decide later if we merge or not`

The above strange because my functions are just calling `bs58DecodeNoCheck(base58String)` and `bs58EncodeNoCheck(data)`

No import

Size :: 1191310
This must be coming from outside my functions.